js/src/forum/components/CommentPost.js

```diff
@@ -131,7 +127,7 @@ export default class CommentPost extends Post {
     const post = this.props.post;
     const props = { post };
 
-    items.add('user', this.postUser.render(), 100);
+    items.add('user', PostUser.component({ post: this.props.post }), 100);
```

Comment:
`{ post: this.props.post }` could be shortened to `{ post }` (itself short for `{ post: post }`) because the value is already available in a variable named `post`

Reply:
Also, there's even that `props` variable there that could be re-used.

But I'd actually suggest dropping that `props` variable because it could introduce unwanted effects to re-use a reference to that variable. `{post}` could simply be used everywhere a `props` variable is used right now.